packages/website/components/Footer.tsx

```diff
@@ -29,11 +29,7 @@ export const Footer = () => {
         <Flex flexDirection="column" marginRight="spacing2Xl">
           <Flex marginRight="spacingL" marginBottom="spacingM">
             <Link href="/playground" passHref>
-              <TextLink
-                href="/playground"
-                variant="secondary"
-                alignIcon="end"
-              >
+              <TextLink href="/playground" variant="secondary" alignIcon="end">
```

Comment:
```suggestion
              <TextLink href="/playground" variant="secondary">
```

There is no icon so no need for this prop